cluster/gce/util.sh

```diff
@@ -745,16 +745,17 @@ function check-cluster() {
       if [[ ${elapsed} -gt ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} ]]; then
           echo -e "${color_red}Cluster failed to initialize within ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} seconds.${color_norm}" >&2
           if [[ ${KUBE_TEST_DEBUG-} =~ ^[yY]$ ]]; then
-            local tmp_log="$(mktemp)"
-            local file
-            for file in /var/log/startupscript.log /var/log/kube-apiserver.log; do
-              echo "${MASTER_NAME}:${file} contents:"
-              if gcloud compute copy-files --project "${PROJECT}" \
-                --zone "${ZONE}" "${MASTER_NAME}:${file}" "${tmp_log}"; then
-                cat "${tmp_log}"
-              fi
+            local logdir=/var/log
+            local basename
+            for basename in startupscript kube-apiserver; do
+              local src="${logdir}/${basename}.log"
+              echo "Preserving ${MASTER_NAME}:${src}"
+              gcloud compute copy-files \
+                -project "${PROJECT}" --zone "${ZONE}" \
+                "${MASTER_NAME}:${src}" \
+                "${E2E_REPORT_DIR}/${basename}.${MASTER_NAME}.log.$(date +'%Y%m%d-%H%M%S')" \
```

Comment:
to be more consistent with what [`test/e2e/core.go` does](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/core.go#L100), maybe swap `${MASTER_NAME}` and `${basename}`?

(We also don't save the datetime there, but I could go either way on that.)